akka-docs/src/main/paradox/scala/project/migration-guide-2.4.x-2.5.x.md

```diff
@@ -526,6 +526,12 @@ They will receive `Replicator.Deleted` instead.
 
 ## Persistence
 
+### `Recovery.none` constant now implies also no snapshot recovery
+
+When using the `Recovery.none` constant to override `PersistentActor#recovery` no recovery at all will be performed,
+that is - also no snapshot recovery will be requested. If you want to skip recovery of events but do want to be offered
```

Comment:
but that was the old behavior also, right? so we are really not changing anything that is worth mentioning in migration guide apart for if someone would use something weird like `Recover.none.copy`